docs/README.md

```diff
@@ -119,6 +122,55 @@ Note:
 
 ---
 
+## Developing a LabVIEW measurement client
+
+1. Create and save a new LabVIEW project.
+
+2. From the project window, go to `Tools` → `Plug-In SDKs` → `Measurements` → `Create Measurement Plug-in Client...`.
+    - In the dialog, select atleast one of the available V2 measurements, then click `Create Measurement Plug-in Client(s)`.
```

Comment:
Consider: "In the dialog, select one or more available measurements,"

Reply:
This should mention that the measurement service(s) must first either be running or deployed on the system. The dialog will only display results for measurements which can be discovered via the discovery service.